.github/CODEOWNERS

```diff
@@ -28,3 +28,7 @@ pkg/images/defaults/**/* @stackrox/maple
 migrator/**/*       @stackrox/data-shepherds
 pkg/postgres/**/*   @stackrox/data-shepherds
 tests/upgrade/*     @stackrox/data-shepherds
+
+.github/**/* @stackrox/circleci
```

Comment:
Should be the new `@stackrox/ci` team now (same for the other two directories) 🙂